tests/improver-estimate-emos-coefficients/02-gaussian.bats

```diff
@@ -36,7 +36,7 @@
   KGO="estimate-emos-coefficients/gaussian/kgo.nc"
 
   # Estimate the EMOS coefficients and check that they match the kgo.
-  run improver estimate-emos-coefficients 'gaussian' '20170605T1500Z' \
+  run improver estimate-emos-coefficients 'gaussian' '20170605T0300Z' \
```

Comment:
Why are these changes needed?

Reply:
This was my mistake when I set up the tests. This time should be the cycletime, see the description in the estimate-emos-coefficients, however, I mistakenly specified the validity time here. I've corrected that here.